front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx

```diff
@@ -0,0 +1,162 @@
+  updatePoint: UpdatePointActions;
+  point: PathStep | null;
+  isPending?: boolean;
+};
+
+const StdcmOperationalPoint = ({ updatePoint, point, isPending }: StdcmOperationalPointProps) => {
+  const dispatch = useAppDispatch();
+
+  const { searchTerm, chCodeFilter, chOptions, sortedResults, setSearchTerm, setChCodeFilter } =
+    useSearchOperationalPoint({ searchTerm: point?.name, chCodeFilter: point?.ch });
+
+  const [selectOptions, setSelectOptions] = useState<string[]>(chOptions);
+
+  const getLabel = ({
+    trigram,
+    name,
+    ci,
+    ch,
+  }: {
+    trigram: string | null;
+    ch: string | null;
+    ci: string | null;
+    name?: string;
+  }) => [trigram, name, ci, ch].join(' ');
+
+  useEffect(() => {
+    let newSelectOptions = [''];
+    if (point?.ch) {
+      newSelectOptions = [...newSelectOptions, point.ch];
+    } else {
+      newSelectOptions = [...newSelectOptions, ...chOptions];
+    }
+    setSelectOptions(newSelectOptions);
+  }, [chOptions, point]);
+
+  useEffect(() => {
+    if (point) {
+      let trigram = '';
+      let ci = '';
+      if ('trigram' in point) {
+        trigram = point.trigram ?? '';
+      }
+      if ('secondary_code' in point) {
+        ci = point.secondary_code || '';
+      }
+
+      setSearchTerm(
+        getLabel({
+          trigram,
+          name: point.name || '',
+          ch: point.ch || '',
+          ci,
+        })
+      );
+      setChCodeFilter(point.ch || '');
+    } else {
+      setSearchTerm('');
+      setChCodeFilter('');
+    }
+  }, [point]);
+
+  const onInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(e.target.value);
+    if (e.target.value.trim().length === 0) {
+      dispatch(updatePoint(null));
+    }
+  };
+
+  const dispatchNewPoint = (p?: SearchResultItemOperationalPoint) => {
+    if (isNil(p)) {
+      dispatch(updatePoint(null));
+      return;
+    }
+    dispatch(
+      updatePoint({
+        secondary_code: p.ci.toString() ?? null,
+        trigram: p.trigram,
+        name: p.name,
+        ch: p.ch,
+        id: p.obj_id,
+        coordinates: p.geographic.coordinates,
+      })
+    );
+  };
+
+  const onSelectSuggestion = ({ value }: StdcmSuggestionsOperationalPointOption) => {
+    dispatchNewPoint(value);
+  };
+
+  const onSelectChCodeFilter = (newchCodeFilter: string = '') => {
+    if (newchCodeFilter.trim().length === 0) {
+      dispatch(updatePoint(null));
+    } else {
+      dispatchNewPoint(sortedResults.find((p) => p.ch === newchCodeFilter));
+    }
+  };
+
+  const operationalPointsSguggestions = sortedResults.map(
+    (p: SearchResultItemOperationalPoint) => ({
+      value: p,
+      label: getLabel({
+        trigram: p.trigram,
+        name: p.name,
+        ch: p.ch,
+        ci: p.ci.toString(),
+      }),
+      ...p,
+    })
+  );
+
+  return (
+    <div className="flex">
+      <div className="col-8">
+        <StdcmSuggestions
+          id="ci"
+          label="CI"
```

Comment:
![Capture d’écran 2024-05-23 à 15 26 53](https://github.com/OpenRailAssociation/osrd/assets/113104137/f636a5eb-4cc1-4814-942b-f533528db420)

The spacing between the label and the selector seems to be bigger than the one of the model (with all inputs): 
![Capture d’écran 2024-05-23 à 15 28 15](https://github.com/OpenRailAssociation/osrd/assets/113104137/862e60a0-010a-4084-8a3b-9652b46de4ed)